disco_aws_automation/resource_helper.py

```diff
@@ -211,3 +204,17 @@ def check_written_s3(object_name, expected_written_length, written_length):
         raise S3WritingError(
             "{0} is not written correctly to S3 bucket".format(object_name)
         )
+
+
+def backoff(cycle):
+    """
+    This function takes as input an integer that represents a cycle count,
+    calculates jitter and executes sleep for the calculated time.
+    The value of 'cycle' must be an integer greater than 0.
+    """
+    base = 3
+    if cycle is None or cycle < 1:
+        raise ValueError('Value of cycle must be int > 0')
+
+    time.sleep(min(MAX_POLL_INTERVAL, randint(base, cycle * 3)))
```

Comment:
it provides acceptable values in most of the runs. here are two examples:
3,4,9,11,4,10,26,34,28,26,40,43,53,16,34,18,32,5
3,5,3,8,21,12,3,4,8,10,22,25,54,62,47,90,23,29,11